libinitializer/GlobalConfigureInitializer.cpp

```diff
@@ -78,6 +90,11 @@ void dev::initializer::initGlobalConfig(const boost::property_tree::ptree& _pt)
     else if (getVersionNumber(version, versionNumber))
     {
         g_BCOSConfig.setSupportedVersion(version, static_cast<VERSION>(versionNumber));
+
+        if (versionNumber < getVersionNumber("2.1.0"))
+        {
+            g_BCOSConfig.setEVMSchedule(dev::eth::FiscoBcosSchedule);

```

Comment:
this should after parsing versionNumber, and set accord g_BCOSConfig.supportedVersion() or g_BCOSConfig.version()

Reply:
done